NDB.Covid19/NDB.Covid19.iOS/Info.plist

```diff
@@ -108,6 +108,6 @@
 	<key>ENDeveloperRegion</key>
 	<string>NO</string>
 	<key>ENAPIVersion</key>
-	<string>1</string>
+	<string>2</string>
```

Comment:
change to integer, string in new nuget gives error

Reply:
Done